services/contract_call_local.proto

```diff
@@ -162,6 +179,11 @@ message ContractFunctionResult {
      * ContractCreateTransactionBody or a ContractCallTransactionBody.     
      */
     AccountID sender_id = 13;
+
+    /**
+    * A list of updated contract account nonces containing the new nonce value for each contract account
```

Comment:
We could note, 
```
This is always empty in the `ContractFunctionResult` field of a `ContractCallLocalResponse`, since no internal creations can happen in a static EVM call.
```

Reply:
Added.